ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -1231,8 +1231,24 @@ func (h *FaultHandler) startNetworkLatencyFault(ctx context.Context, taskMetadat
 		field.CommandOutput: string(cmdOutput[:]),
 	})
 	// After creating the queueing discipline, create filters to associate the IPs in the request with the handle.
+	// First redirect the allowlisted ip addresses to band 1:3 where is no network impairments.
+	for _, ip := range request.SourcesToFilter {
+		tcAllowlistIPCommandComposed := nsenterPrefix + fmt.Sprintf(tcAllowlistIPCommandString, interfaceName, aws.StringValue(ip))
+		cmdList = strings.Split(tcAllowlistIPCommandComposed, " ")
+		_, err = h.runExecCommand(ctx, cmdList)
+		if err != nil {
+			logger.Error("Command execution failed", logger.Fields{
+				field.CommandString: tcAllowlistIPCommandComposed,
+				field.Error:         err,
+				field.CommandOutput: string(cmdOutput[:]),
+				field.TaskARN:       taskMetadata.TaskARN,
+			})
+			return err
```

Comment:
Just confirming - this error does not get added to the response sent back to the client, right?

Reply:
The Linux command output won't be added to the response, but we will add the err to the response